client/client-multi/client-c/src/kaa_status.c

```diff
@@ -13,7 +13,11 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
+#include <stddef.h>
+#include <stdint.h>
+#include <stdbool.h>
+#include "platform/stdio.h"
+#include "platform/sha.h"
```

Comment:
Are you sure you absolutely need every of these includes here? Let us please not include anything that is not absolutely necessary.